lib/src/main/java/com/jesusm/holocircleseekbar/lib/HoloCircleSeekBar.java

```diff
@@ -1,21 +1,5 @@
-/*
```

Comment:
This shouldn't be removed as it indicates that this code was initially created by Lars Werkman.